addons/board/models/board.py

```diff
@@ -34,10 +34,16 @@ def get_view(self, view_id=None, view_type='form', **options):
                         'arch': custom_view.arch})
         res.update({
             'arch': self._arch_preprocessing(res['arch']),
-            'toolbar': {'print': [], 'action': [], 'relate': []}
         })
```

Comment:
Maybe update this to be less crap while at it?
```suggestion
res['arch'] = self._arch_preprocessing(res['arch'])
```